### PR TITLE
OS#15710484 Load null(0) instead of RecyclableObject null object for stack args optimization

### PR DESCRIPTION
**Pull request by agarwal-sandeep**

### CODE REVIEW

**Comment by agarwal-sandeep** [comment]
@pleath @rajatd please take a look

**Comment by agarwal-sandeep** [comment]
@pleath @rajatd ping.

File: lib/Runtime/Language/JavascriptOperators.cpp [hunk 1]
```diff
@@ -5573,8 +5573,13 @@ namespace Js
     void JavascriptOperators::OP_InvalidateCachedScope(void* varEnv, int32 envIndex)
     {
         FrameDisplay *disp = (FrameDisplay*)varEnv;
-        RecyclableObject *objScope = RecyclableObject::FromVar(disp->GetItem(envIndex));
-        objScope->InvalidateCachedScope();
+        Var item = disp->GetItem(envIndex);
```
Comment (pleath) [inline lib/Runtime/Language/JavascriptOperators.cpp]: Under what condition do we actually have a null ptr taking up space in the FrameDisplay?

Comment (agarwal-sandeep) [reply lib/Runtime/Language/JavascriptOperators.cpp]: This used to be Null RecyclableObject object and RecyclableObject::InvalidateCachedScope is a no-op. 

LowerLdFrameDisplay copies the content of environment which can copy nullptr if we  did stackarg optimization.

Exprgen generate tons of such cases.

**Comment by agarwal-sandeep** [comment]
@LouisLaf ping

File: lib/Runtime/Language/JavascriptOperators.cpp (hunk 1 above)
Comment (pleath) [reply lib/Runtime/Language/JavascriptOperators.cpp]: That raises the question: why do we access the null slot in the FrameDisplay? You'd think we would only do that in the presence of eval, but if there's eval we shouldn't be doing stack args.

**Pull request merged**